tests/test_dbt_project/dbt_sugar_test/models/folder_to_exclude/schema.yml

```diff
@@ -1 +1 @@
-version: 1
```

Comment:
dbt was giving a compilation error:
Compilation Error                                                                                                                         
                      The schema file at folder_to_exclude/schema.yml is invalid because version 1 is not supported. Please consult the documentation for more
                    information on schema.yml syntax:

Reply:
I also needed to delete the sql model in target because dbt was complaining that they were 2.

Reply:
oh yeah good catch